### PR TITLE
pull request 1

### PR DESCRIPTION
**Pull request by Katta-Tejaswi**
added some txt to dev2 file

### CODE REVIEW

File: dev1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+file1
```
Comment (bhanup94) [inline dev1.txt]: update this line

File: dev2.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+dev file2
```
Comment (bhanup94) [inline dev2.txt]: update

File: dev1.txt (hunk 1 above)
Comment (Katta-Tejaswi) [reply dev1.txt]: updated https://github.com/Katta-Tejaswi/Branch-Test/pull/1/commits/07d1c955d007174b2da3750dafdecf3e60ec3eb8

File: dev2.txt (hunk 2 above)
Comment (Katta-Tejaswi) [reply dev2.txt]: updated https://github.com/Katta-Tejaswi/Branch-Test/pull/1/commits/07d1c955d007174b2da3750dafdecf3e60ec3eb8

**Review by bhanup94 — APPROVED** [review]

**Pull request merged**